api/restapi/configure_keptn.go

```diff
@@ -56,7 +62,7 @@ func configureAPI(api *operations.KeptnAPI) http.Handler {
 		os.Exit(1)
 	}
 
-	/// configure the api here
+	// / configure the api here
```

Comment:
That looks like an accidental space :)

Reply:
This is coming from auto-formatting of the code once I touched the file (some comments are not well formatted and somlines are a little long). Undid the format changes for the unaffected parts of the file and pushed a fixup